src/renderer/scss/component/_snack-bar.scss

```diff
@@ -5,16 +5,15 @@
   color: var(--snackbar-color-primary);
   display: flex;
   justify-content: space-between;
-  left: 0;
   margin-left: auto;
   margin-right: auto;
   max-width: var(--snack-bar-width);
   min-width: 300px;
   opacity: 0.95;
   padding: 14px 20px 10px 20px;
   position: fixed;
-  right: 0;
-  top: $spacing-vertical;
+  right: $spacing-vertical;
```

Comment:
Should it be `bottom-left` or `bottom-right` ?
I think in the right looks good :thinking:

Reply:
I think I like the left side more